### DISCUSSION TITLE
I only need Quasar's components, I don't need any of the other features. Can I safely ditch the CLI?

### DISCUSSION THREAD

**Discussion by mklsh** [quasarframework/quasar#13782] [General - Components / Directives / etc]
Basically what the title says. I don't care about the different modes or the project structure that Quasar provides. I only want the components. Can I safely ditch the CLI and use Quasar as a Vue plugin?

**Comment by metalsadman** [comment]
https://quasar.dev/start/pick-quasar-flavour#introduction 
read the warning
https://quasar.dev/start/vue-cli-plugin#introduction
